### ISSUE TITLE
Is this working with iPad devices or iPad simulator ?

### ISSUE THREAD

**Issue by isurugajasinghe** [beefe/react-native-actionsheet#51]

**Comment by alexlevy0** [comment]
@isurugajasinghe Yes both

**Comment by naytun** [comment]
No, it is not working properly on iPad. 'Cancel' button is not showing up.

<img width="241" alt="image" src="https://user-images.githubusercontent.com/10265410/69903962-70526600-136e-11ea-99bd-a4ce3ee6c0fb.png">
